### PR TITLE
HDDS-8038. [Snapshot] Add a smoketest for Snapshot with ACLs

### PR DESCRIPTION
**Pull request by mladjan-gadzic**
## What changes were proposed in this pull request?
Add smoketest for Snapshot with ACLs.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-8038

## How was this patch tested?
Smoketest.

### CODE REVIEW

**Comment by GeorgeJahad** [comment]
LGTM

File: hadoop-ozone/dist/src/main/smoketest/snapshot/snapshot-acls.robot [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+${USER2} =              testuser2
+${KEY}
+${FIRST_SNAPSHOT}
+${SECOND_SNAPSHOT}
+
+*** Keywords ***
+Add ACL
+    [arguments]         ${object}   ${user}     ${objectName}
+    ${result} =         Execute     ozone sh ${object} addacl -a user:${user}/scm@EXAMPLE.COM:a ${objectName}
+    Should not contain  ${result}   Failed
```
Comment (aswinshakil) [inline hadoop-ozone/dist/src/main/smoketest/snapshot/snapshot-acls.robot:34]: The general error message doesn't have `Failed` in it, So even if it failed we can't be sure. For example here 
```
PERMISSION_DENIED User scm/scm@EXAMPLE.COM doesn't have WRITE_ACL permission to access volume Volume:s3v
```
Suggestion would be change it to `PERMISSION_DENIED`

**Review by aswinshakil — COMMENTED** [review]
Thanks for the patch @mladjan-gadzic. Overall the change looks good to me. I just have one comment inline.

**Comment by mladjan-gadzic** [comment]
@GeorgeJahad @aswinshakil thank you for the review!

**Review by prashantpogde — APPROVED** [review]
LGTM once the requested changes are done!

**Review by aswinshakil — APPROVED** [review]
Thanks for the patch @mladjan-gadzic. LGTM+1

**Pull request merged**